lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -712,17 +712,6 @@ private void calculateOffsets(final float fraction) {
       textPaint.setColor(getCurrentCollapsedTextColor());
     }
 
-    if (collapsedLetterSpacing != expandedLetterSpacing) {
-      textPaint.setLetterSpacing(
-          lerp(
-              expandedLetterSpacing,
-              collapsedLetterSpacing,
-              fraction,
-              AnimationUtils.FAST_OUT_SLOW_IN_INTERPOLATOR));
-    } else {
-      textPaint.setLetterSpacing(collapsedLetterSpacing);
-    }
-
```

Comment:
Instead of removing this block, could it be moved into the `else` statement of `if (fadeModeEnabled)`?

```
if (fadeModeEnabled) {
   ...
} else {
   lerp letter spacing
}
```